helm/icees-api/templates/redis-deployment.yaml

```diff
@@ -0,0 +1,59 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ include "icees-api.fullname" . }}-trapi-{{ .Chart.AppVersion }}-redis
+  labels:
+    app.kubernetes.io/name: {{ include "icees-api.fullname" . }}-trapi-{{ .Chart.AppVersion }}-redis
+    app.kubernetes.io/instance: {{ include "icees-api.fullname" . }}-trapi-{{ .Chart.AppVersion }}-redis
+    server-type: redis
+    trapi: "1.2"
+spec:
+  replicas: {{ .Values.replicaCount }}
+  strategy:
+    type: Recreate
+  selector:
+    matchLabels:
+      app.kubernetes.io/name: {{ include "icees-api.fullname" . }}-trapi-{{ .Chart.AppVersion }}-redis
+      app.kubernetes.io/instance: {{ include "icees-api.fullname" . }}-trapi-{{ .Chart.AppVersion }}-redis
+      server-type: redis
+      trapi: "1.2"      
+  template:
+    metadata:
+      labels:
+        app.kubernetes.io/name: {{ include "icees-api.fullname" . }}-trapi-{{ .Chart.AppVersion }}-redis
+        app.kubernetes.io/instance: {{ include "icees-api.fullname" . }}-trapi-{{ .Chart.AppVersion }}-redis
+        server-type: redis
+        trapi: "1.2"      
+    spec:
+      containers:
+        - name: {{ include "icees-api.fullname" . }}-trapi-{{ .Chart.AppVersion }}-redis
+          image: redis:6.2
```

Comment:
please do not hard code image versions. they are better placed in a values file.